Networking/NetworkingTests/Mapper/SitePluginMapperTests.swift

```diff
@@ -26,6 +26,26 @@ final class SitePluginMapperTests: XCTestCase {
         XCTAssertEqual(plugin?.version, "9.5")
         XCTAssertEqual(plugin?.textDomain, "jetpack")
     }
+
+    /// Verifies the SitePlugin fields are parsed correctly when there's no data envelope wrapping the response.
+    ///
+    func test_SitePlugin_fields_are_properly_parsed_for_response_without_data_envelope() {
+        let plugin = mapPluginWithoutEnvelope(from: "site-plugin-without-envelope")
+        XCTAssertNotNil(plugin)
```

Comment:
nit: it might be nice avoiding the optionals by unwrapping it early

```suggestion
    func test_SitePlugin_fields_are_properly_parsed_for_response_without_data_envelope() throws {
        let plugin = try XCTUnwrap(mapPluginWithoutEnvelope(from: "site-plugin-without-envelope"))
```

Reply:
Updated in 387a766d50797e99f52e8fce5c6f7568254a075c.